quickstart/101-managed-instance/main.tf

```diff
@@ -0,0 +1,85 @@
+# Create resource group
+resource "azurerm_resource_group" "example" {
+  name     = var.azurerm_resource_group_name
+  location = var.location
+}
+
+# Create security group
+resource "azurerm_network_security_group" "example" {
+  name                = var.azurerm_network_security_group_name
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+# Create a virtual network
+resource "azurerm_virtual_network" "example" {
+  name                = var.azurerm_virtual_network_name
+  resource_group_name = azurerm_resource_group.example.name
+  address_space       = ["10.0.0.0/16"]
+  location            = azurerm_resource_group.example.location
+}
+
+# Create a subnet
+resource "azurerm_subnet" "example" {
+  name                 = var.azurerm_subnet_name
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefixes     = ["10.0.0.0/24"]
```

Comment:
Same as above, why such a big subnet? could you use /27 instead?

Reply:
done